WordPress/Classes/ViewRelated/Reader/ReaderTableContent.swift

```diff
@@ -9,13 +9,24 @@ final class ReaderTableContent {
         tableViewHandler?.delegate = delegate
     }
 
+    func resetResultsController() {
+        tableViewHandler?.resetResultsController()
+        tableViewHandler?.tableView.reloadData()
+        tableViewHandler?.tableView.layoutIfNeeded()
+
+        if !isEmpty {
+            tableViewHandler?.tableView.scrollToRow(at: IndexPath(row: 0, section: 0), at: .top, animated: false)
```

Comment:
I had to use `scrollToRow` here because using `setContentOffset` would scroll too high sometimes above the first card. I believe this was due to the pull-to-refresh.